core/blockchain.go

```diff
@@ -1360,10 +1361,14 @@ func (bc *BlockChain) writeBlockWithState(block *types.Block, receipts []*types.
 		log.Crit("Failed to write block into disk", "err", err)
 	}
 	// Commit all cached state changes into underlying memory database.
-	root, err := state.Commit(bc.chainConfig.IsEIP158(block.Number()))
-	if err != nil {
-		return NonStatTy, err
+	root, stateChanges, commitErr := state.Commit(bc.chainConfig.IsEIP158(block.Number()))
+	log.Debug("Sending StateChangeEvent to the feed", "block number", block.Number(), "count", len(stateChanges))
+	bc.stateChangeEventFeed.Send(StateChangeEvent{block, stateChanges})
```

Comment:
Also, looks like the `logsFeed` has a send [in the `reorg` function](https://github.com/ethereum/go-ethereum/blob/a75c0610b7aad63de01cefeeaf3426b332e718eb/core/blockchain.go#L2087) - I wonder if we might want to be sending from there as well. Maybe worth kicking off to a separate story though 🤔

Reply:
Added a story to account for this: https://makerdao.atlassian.net/browse/VDB-1274